CHANGELOG.md

```diff
@@ -4,6 +4,28 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 5.15.0
+### Added
+* Service: Use `InterpolatedStringHandler` to move values to `CustomDimensions` Tags [#2450](https://github.com/microsoft/onefuzz/pull/2450)
+* Service: C# Can create ADO notifications [#2456](https://github.com/microsoft/onefuzz/pull/2456), [#2458](https://github.com/microsoft/onefuzz/pull/2458)
+* ```
+* Service: C# Cache VMSS VM InstanceID lookups [#2464](https://github.com/microsoft/onefuzz/pull/2464)
+* CLI: Retry on connection reset [#2468](https://github.com/microsoft/onefuzz/pull/2468)
+* Agent: Enable backtraces for agent errors [#2437](https://github.com/microsoft/onefuzz/pull/2437)
+### Changed
+* Service: Bump Dependencies [#2446](https://github.com/microsoft/onefuzz/pull/2446)
+* Service: Temporarily disable Pool validation [#2459](https://github.com/microsoft/onefuzz/pull/2459)
+* Service: Added `New` to Node State Machine. [#2476](https://github.com/microsoft/onefuzz/pull/2476)
```

Comment:
I believe this PR was revised to _not_ introduce a `New` state.

Reply:
You're right. Fixing.

Reply:
@stishkin, could you confirm what that PR ended up doing, since your inline comments don't fully apply after your revision? What should @nharper285 write for it? Just skimming, it's no longer clear to me if you changed the state machine behavior or not.

Reply:
Confirming: I reverted the "new" state

Reply:
Did you change the state machine in any other way, or just add logging?

Reply:
just added logs

Reply:
Hm suggestion for reasonable PR detail? Seems like 'Fixed State Machine' no longer really applies?

Reply:
no longer applies, that's correct

Reply:
Suggestion on PR detail?

Reply:
well actually it does apply, since "SetState" on the Node was fixed, because C# had different implementation from Python, and I made them same